app/models/petition.rb

```diff
@@ -935,9 +926,9 @@ def close!(time)
       raise RuntimeError, "can't close a petition that is in the #{state} state"
     end
 
-    if deadline <= time
+    if past_deadline?(time)
       Appsignal.increment_counter("petition.closed", 1)
-      update!(state: CLOSED_STATE, closed_at: deadline)
+      update!(state: CLOSED_STATE, closed_at: (deadline || time))
```

Comment:
This check is to prevent a petition being closed early - since there's no definition of early now then we can call the `close_early!` method `close!` and remove this method. We can also remove the use of `deadline` throughout the app.